### PR TITLE
Add 2.0.0.beta2 announcement

### PR DESCRIPTION
**Pull request by timriley**
[readable](https://github.com/hanami/hanami.github.io/blob/announcing-hanami-2.0.0.beta2/source/blog/2022-08-16-announcing-hanami-200beta2.html.markdown)

### CODE REVIEW

File: source/blog/2022-08-16-announcing-hanami-200beta2.html.markdown [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+# config/app.rb
+
+module AppName
+  class App < Hanami::App
+    config.slices = %w[blog shop]
+  end
+end
+```
+
+You can also populate this setting direectly via an `HANAMI_SLICES` environment variable, using commas to separate the slice names, e.g. `HANAMI_SLICES=blog,shop`.
```
Comment (solnic) [inline source/blog/2022-08-16-announcing-hanami-200beta2.html.markdown]:
```suggestion
You can also populate this setting directly via an `HANAMI_SLICES` environment variable, using commas to separate the slice names, e.g. `HANAMI_SLICES=blog,shop`.
```

File: source/blog/2022-08-16-announcing-hanami-200beta2.html.markdown [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+Any slices not incuded in this list will be completely ignored: their slice class will not be loaded, nor any of their other Ruby source files; effectively, they will not exist. Specifying slices like this will improve boot time and minimize memory usage for specific workloads (imagine a job runner that needs only a single slice), as well as help ensure clean boundaries between your slices.
```
Comment (solnic) [inline source/blog/2022-08-16-announcing-hanami-200beta2.html.markdown]:
```suggestion
Any slices not incuded in this list will be completely ignored: their slice class will not be loaded, nor any of their other Ruby source files; effectively, they will not exist. Specifying slices like this **will improve boot time and minimize memory usage** for specific workloads (imagine a job runner that needs only a single slice), as well as help ensure clean boundaries between your slices.
```

**Review by solnic — APPROVED** [review]
left a couple of suggestions

File: source/blog/2022-08-16-announcing-hanami-200beta2.html.markdown (hunk 2 above)
Comment (timriley) [reply source/blog/2022-08-16-announcing-hanami-200beta2.html.markdown]: Nice :) Have done that while also making that sentence a dedicated paragraph.

**Pull request merged**